store/rootmulti/snapshot_test.go

```diff
@@ -128,7 +128,7 @@ func TestMultistoreSnapshot_Checksum(t *testing.T) {
 			"aa048b4ee0f484965d7b3b06822cf0772cdcaad02f3b1b9055e69f2cb365ef3c",
 			"7921eaa3ed4921341e504d9308a9877986a879fe216a099c86e8db66fcba4c63",
 			"a4a864e6c02c9fca5837ec80dc84f650b25276ed7e4820cf7516ced9f9901b86",
-			"ca2879ac6e7205d257440131ba7e72bef784cd61642e32b847729e543c1928b9",
+			"8ca5b957e36fa13e704c31494649b2a74305148d70d70f0f26dee066b615c1d0",
```

Comment:
digging through the code and finding it hard to see why this changed. I thought because maybe {} was being included in the checksum but doesn't seem so

Reply:
No idea why this change caused this test to fail. Maybe just update the test assertion?

Reply:
I suspect you are using go 1.19 right? https://github.com/cosmos/cosmos-sdk/actions/runs/3097837960/jobs/5015002343